.travis.yml

```diff
@@ -1,20 +1,12 @@
-sudo: required
-dist: bionic
-language: generic
-install: git clone https://github.com/QubesOS/qubes-builder ~/qubes-builder
+import:
+  - source: QubesOS/qubes-continuous-integration:R4.1/travis-base-r4.1.yml
 script: 
- - ~/qubes-builder/scripts/travis-build
```

Comment:
A side effect of travis-build script was copying (actually - hardlinking) installer-qubes-os into ~/qubes-builder. See travis failure now.